extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -53,10 +53,10 @@
    z-index: 2147483646;
    width: 100%;
    position: fixed;
-   top: 0;
```

Comment:
top: 0 breaks the place holder, and doesn't keep it within it's container. Therefore the multiple toolbars will over lap if they all have the same top value.